idoit-install

```diff
@@ -51,7 +51,8 @@ IFS=$'\n\t'
 : "${JOBS_BIN:="/usr/local/bin/idoit-jobs"}"
 : "${CRON_FILE:="/etc/cron.d/i-doit"}"
 : "${BACKUP_DIR:="/var/backups/i-doit"}"
-: "${RECOMMENDED_PHP_VERSION:="7.3"}"
+: "${RECOMMENDED_PHP_VERSION:="7.4"}"
+: "${RECOMMENDED_MARIADB_VERSION:="10.4"}"
```

Comment:
Should be `10.5` :wink:

Reply:
Yeah, the change was done some months ago  :-) 
I will create a new updated PR for all OS and app updates ;-)